src/main/kotlin/no/nav/syfo/dialogmelding/bestilling/domain/DialogmeldingToBehandlerBestilling.kt

```diff
@@ -15,6 +15,7 @@ data class DialogmeldingToBehandlerBestilling(
     val kode: DialogmeldingKode,
     val tekst: String?,
     val vedlegg: ByteArray? = null,
+    val kilde: String?,
```

Comment:
Gir det mening å ha et kodeverk/enum her? Sånn at vi har kontroll på alle kildene?
Det gjør jo at vi må lage en PR og oppdatere kilde hver gang det kommer en ny produsent, men det er kanskje også en fin type "kontroll"?

Da kan vi feks bestemme hvilket nivå et kilde-navn skal være.

Reply:
Ja, det er kanskje lurt. Men vi leser jo disse inn fra et Kafka-topic - hva skal vi gjøre hvis det kommer inn en record med en ukjent verdi? (dette er jo bare et felt til informasjon - det har ikke noe betydning for om det går bra å sende dialogmeldingen eller ikke)

Reply:
Usikker på om det er lurt med enum :thinking: Tenker i utgangspunktet det burde være tilstrekkelig med tilgangskontrollen vi allerede har på topicet samt at vi gjør kilde-feltet påkrevd og har noe dokumentasjon på hva vi forventer kilde-feltet brukes til :man_shrugging:

Reply:
Så ikke kommentaren før nå. Er ikke Enum den enkleste formen for dokumentasjon i så fall?
Eventuelle nye konsumenter vil måtte innrette seg vårt format og kan enkelt finne det i koden uten å måtte lete 🤷🏼‍♂️